go/teams/service_helper.go

```diff
@@ -75,7 +75,10 @@ func GetAnnotatedTeam(ctx context.Context, g *libkb.GlobalContext, teamID keybas
 	}
 	var joinRequests []keybase1.TeamJoinRequest
 	var tarsDisabled bool
-	if myRole.IsOrAbove(keybase1.TeamRole_ADMIN) {
+	if myRole == keybase1.TeamRole_NONE || myRole.IsOrAbove(keybase1.TeamRole_ADMIN) {
```

Comment:
Should we do `team.getAdminPermission` to be sure we really are an implicit admin? I don't think that's necessary but this code seems scary. `myRole == NONE` seems like a neat trick for detecting implicit admins

Reply:
This should be less scary now with AppStatus checking and not bailing from entire call if it's "just" a permission error.